Essentials/build.gradle

```
@@ -41,7 +42,7 @@ shadowJar {
         include (project(':providers:1_8Provider'))
     }
     relocate 'io.papermc.lib', 'com.earth2me.essentials.paperlib'
-    relocate 'org.bstats.bukkit', 'com.earth2me.essentials.metrics'
+    relocate 'org.bstats', 'com.earth2me.essentials.metrics'
```

May as well change this to match the newer shaded libraries while we're at it:

```suggestion
    relocate 'org.bstats', 'com.earth2me.essentials.libs.bstats'
```